fides-aws-ecs/config/privacyCenterConfig.json

```
@@ -0,0 +1,100 @@
+      "title": "Access your data",
+      "description": "We will provide you a report of all your personal data.",
+      "description_subtext": [],
+      "identity_inputs": {
+        "email": "required"
+      },
+      "confirmButtonText": "Continue",
+      "cancelButtonText": "Cancel"
+    },
+    {
+      "policy_key": "default_erasure_policy",
+      "icon_path": "/delete.svg",
+      "title": "Erase your data",
+      "description": "We will erase all of your personal data. This action cannot be undone.",
+      "description_subtext": [],
+      "identity_inputs": {
+        "email": "required"
+      },
+      "confirmButtonText": "Continue",
+      "cancelButtonText": "Cancel"
+    }
+  ],
+  "includeConsent": true,
+  "consent": {
+    "button": {
+      "description": "Manage your consent preferences, including the option to select 'Do Not Sell or Share My Personal Information'.",
+      "description_subtext": [
+        "In order to opt-out of certain consent preferences, we may need to identify your account via your email address. This is optional."
+      ],
+      "icon_path": "/consent.svg",
+      "identity_inputs": {
+        "email": "optional"
+      },
+      "title": "Manage your consent",
+      "modalTitle": "Manage your consent",
+      "confirmButtonText": "Continue",
+      "cancelButtonText": "Cancel"
+    },
+    "page": {
+      "consentOptions": [
+        {
+          "fidesDataUseKey": "marketing.advertising",
+          "name": "Data Sales or Sharing",
+          "description": "We may use some of your personal information for behavioral advertising purposes, which may be interpreted as 'Data Sales' or 'Data Sharing' under regulations such as CCPA, CPRA, VCDPA, etc.",
+          "url": "https://example.com/privacy#data-sales",
+          "default": {
+            "value": true,
+            "globalPrivacyControl": false
+          },
+          "highlight": false,
+          "cookieKeys": ["data_sales"],
+          "executable": false
+        },
+        {
+          "fidesDataUseKey": "marketing.advertising.first_party",
+          "name": "Email Marketing",
+          "description": "We may use some of your personal information to contact you about our products & services.",
+          "url": "https://example.com/privacy#email-marketing",
+          "default": {
+            "value": true,
+            "globalPrivacyControl": false
+          },
+          "highlight": false,
+          "cookieKeys": ["tracking"],
+          "executable": false
+        },
+        {
+          "fidesDataUseKey": "functional",
+          "name": "Product Analytics",
+          "description": "We may use some of your personal information to collect analytics about how you use our products & services.",
+          "url": "https://example.com/privacy#analytics",
+          "default": true,
+          "highlight": false,
+          "cookieKeys": ["analytics"],
+          "executable": false
+        }
+      ],
```

```suggestion
      "consentOptions": [],
```

We should change this default to not include legacy static consent options 👍